index.html

```diff
@@ -103,7 +103,7 @@ <h1>
     </p>
     <p>
       You will find most of these apps in the <a href="https://f-droid.org/" target="_blank">F-Droid</a> repository.
-      Also, you might want to consider flashing a custom ROM like <a href="https://www.cyanogenmod.org/" target="_blank">CyanogenMod</a> or <a href="https://www.replicant.us/" target="_blank">Replicant</a> on your device.
```

Comment:
Looks like you removed one of the double quotes (`"`) around the `href` here. Could you add back?

Reply:
oops, fixed now